src/gmt_keywords.txt

```diff
@@ -176,3 +176,6 @@ TIME_UNIT			# Unit of time measurement
 TIME_SYSTEM			# Shortcut to setting TIME_EPOCH/UNIT [does not go in gmt.conf]
 TIME_WEEK_START			# Day a new week starts on
 TIME_Y2K_OFFSET_YEAR		# For 2-digit years to separate 1900 and 2000 centuries
+#-------------------------------------------------------
+SET_RUN_MODE		# To change between CLASSIC and MODERN mode. Used only by externals.
```

Comment:
What's the use case of the new keyword? Is `GMT_RUN_MODE` a better name?

Reply:
OK, renamed to GMT_RUN_MODE.

The use case, as mentioned in the comment, is to allow externals change to MODERN (and back if wished) mode. I need these type of manipulations in GMT.jl to use the autoscaling of the modern them while in a session whose API started as CLASSIC. GMT.jl uses almost only classic mode but some times it needs features only available in modern. For example, I have a working prototype of using the _inset_ command from within a classic session. This keyword makes things simpler.